components/InlineCarousel/src/index.scss

```diff
@@ -81,11 +90,9 @@
   justify-content: var(--denhaag-inline-carousel-content-download-justify-content);
   align-items: var(--denhaag-inline-carousel-content-download-align-items);
   background-color: var(--denhaag-inline-carousel-content-download-background-color);
-  transition: var(--denhaag-inline-carousel-content-download-transition);
 
   &:hover {
     cursor: var(--denhaag-inline-carousel-content-download-hover-cursor);
```

Comment:
Dit zou toch al default gezet kunnen worden of reageert hij op de :hover anders dan de default?

Waarde mag hard gezet worden -> a11y heeft hier bepaalde eisen over, denk niet dat we dit gaan overschrijven met een andere CSS value.

Reply:
Moeten we niet alles in tokens zetten?